biogeochem/EDPhysiologyMod.F90

```diff
@@ -1291,17 +1266,11 @@ subroutine phenology_leafonoff(currentSite)
                    ! This sets the cohort to the "leaves off" flag
                    currentCohort%status_coh      = leaves_off
 
-                   ! Remember what the leaf mass was for next year
-                   currentCohort%leafmemory   = leaf_c
-
                    call PRTDeciduousTurnover(currentCohort%prt,ipft, &
                         leaf_organ, leaf_drop_fraction)
 
                    if(prt_params%woody(ipft).ne.itrue)then
 
-                      currentCohort%sapwmemory   = sapw_c * stem_drop_fraction
-                      currentCohort%structmemory   = struct_c * stem_drop_fraction
-
```

Comment:
Another good reason to drop the memory variables: leafmemory was the prior leaf biomass, whereas sapwmemory/structmemory were the amount of woody biomass lost. I actually had changed the definition of sapwmemory/structmemory in #801 to the same as leafmemory because I found the original definitions very confusing...

Reply:
i saw that too, not sure if this will come up on tests, but glad its correct now